nservicebus/operations/opentelemetry.md

```diff
@@ -3,10 +3,75 @@ title: OpenTelemetry
 summary: Observability of NServiceBus endpoints with OpenTelemetry
 component: core
 reviewed: 2022-06-29
+versions: '[8,)'
 related:
  - samples/open-telemetry
 ---
 
 NServiceBus version 8 and above supports [OpenTelemetry](https://opentelemetry.io/docs/instrumentation/net/) through traces, metrics, and logging.
 
-partial: content
\ No newline at end of file
+Enable OpenTelemetry instrumentation in NServiceBus:
```

Comment:
Why was this inlined? I just split this up further in another PR that conflicts with this change, so we'll have to coordinate on which structure to keep. Given what I've added now in the tracing section, I think it's getting large enough to benefit from splitting up. See https://github.com/Particular/docs.particular.net/pull/6718

Reply:
Laila, go ahead with that PR. Once that's merged, I'll rebase (or maybe even recreate) this one to align with those larger changes. No worries.